gap/Magic.gi

```diff
@@ -519,7 +519,11 @@ function( arg )
         # of the documentation are also in UTF-8 encoding, and may contain characters
         # not contained in the default Latin 1 encoding.
         SetGapDocLaTeXOptions( "utf8", gapdoc_latex_option_record );
-
+        
+        ## HACK: If there is an empty index, MakeGAPDocDoc throws an error when creating the pdf.
+        ## this addition prevents this by fake adding the index to the page number log.
+        Append( GAPDoc2LaTeXProcs.Tail, "\\immediate\\write\\pagenrlog{[\"Ind\", 0, 0], \\arabic{page},}\n" );
```

Comment:
I'd mention #106 in the comment, e.g. `See issue #106` -- this way, one can quickly look up details.